### DISCUSSION TITLE
Week 1 - DevOps Overview

### DISCUSSION THREAD

**Discussion by MichaelCade** [MichaelCade/90DaysOfDevOps#3] [General]
During the first week or first 6 days of the challenge, we are going to be exploring the foundational and overview of DevOps. The reason for this discussion is to allow the community to add additional ideas or topics that should be covered or it could be also used for feedback of this week.

**Comment by PreacherCmdLets** [comment]
like to know how do you start your work? , do you make separate notes before you post your final comment in Repo?

**Reply by MichaelCade** [reply]
I generally have a flow that I want to tackle for the section (7 day topic) so will watch the resources and take notes straight into VSCode. I might then go back and take some screenshots to help walkthrough my thinking and working. 

On the Go Programming I would say watching the larger resource sets and playlists I have shared will give you a good foundational knowledge of Go. I am not going to be able to cover all aspects in writing I just take some of the findings and document them. 

Great question and thanks for getting involved.

**Comment by PeixiZ** [comment]
Hi this is my first day to view this repo, but i can't open the youtube video link that provided below the 01day readme(Chinese block). Can author provide another link that can download ? thanks~

**Reply by PeixiZ** [reply]
> Hi this is my first day to view this repo, but i can't open the youtube video link that provided below the 01day readme(Chinese block). Can author provide another link that can download ? thanks~

always say welcome your PR, but not welcome your network question? pls stop hypothesis that the video in Youtube all of us can access,ok?

**Reply by MichaelCade** [reply]
Apologies on this, I don't seem to get notified on discussion topics! Appreciate you are probably not following along. I also appreciate that some regions will not be able to access the content and unfortunately I cannot or do not have a good way or time to test this. 

We do have active contributors to the translation of the repository to different languages, my advice would be to locate those contributors via an issue and raise this concern, at least this way you can share feedback at worst we can remove dead links that do not work for you in your country but maybe they can help with finding new resources. 

Once again I am sorry for not communicating back sooner.
